indica/equilibrium.py

```diff
@@ -533,21 +542,26 @@ def flux_coords(
             If ``t`` was not specified as an argument, return the time the
             results are given for. Otherwise return the argument.
         """
-        _R, _z = prepare_coords(
-            R + np.full_like(R, self.R_offset), z + np.full_like(z, self.z_offset)
-        )
+        _R = R + self.R_offset
+        _z = z + self.z_offset
+
         if t is not None:
             check_time_present(t, self.t)
             rho = self.rho.interp(t=t, method="nearest")
             R_ax = self.rmag.interp(t=t, method="nearest")
             z_ax = self.zmag.interp(t=t, method="nearest")
             z_x_point = self.zx.interp(t=t, method="nearest")
+            _z = _z.interp(t=t, kwargs={"fill_value": 0})
```

Comment:
you shouldn't need the "fill_value" since the interpolation should not work if the time requested is not present in self.t, so you can just call it as `.interp(t=t)`